playbooks/roles/certbot/tasks/main.yml

```diff
@@ -43,6 +43,17 @@
     path: /var/www/certbot
     state: directory
 
+- name: Create Certbot deploy hooks directory
+  file:
+    path: /etc/letsencrypt/renewal-hooks/deploy
+    state: directory
+
+- name: Copy the Certbot deploy hook to restart nginx on renewal
+  copy:
+    src: restart-nginx.sh
+    dest: /etc/letsencrypt/renewal-hooks/deploy/restart-nginx.sh
+    mode: 755
```

Comment:
This needs to be an octal number, so you need to prefix it with a zero.  (This is a rather unexpected pitfall.  When using the old style, it's automatically treated as octal by Ansible.  With the new style, the number is interpreted by the YAML parser instead of the Ansible code, so it's parsed according to the YAML rules.)

Reply:
@smarnach, I wasn't aware of this behaviour till now. Will fix it.